.env_config_sample

```diff
@@ -8,4 +8,7 @@ SOCIAL_USER_PASS = 'SpartaAuthorsHaven2019'
 EMAIL_HOST_USER = 'your email'
 EMAIL_HOST_PASSWORD = 'your password'
 EMAIL_HOST='smtp.gmail.com'
-EMAIL_PORT='587'
+EMAIL_PORT='587
```

Comment:
@felixkiryowa kindly avoid introducing unnecessary. Kindly restore the missing quotation mark on line 11